test/e2e/chainsaw/check-multi-resource-rgd/instance-create-assert.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   generation: 1
   labels:
     kro.run/owned: "true"
-    kro.run/resource-graph-definition-name: check-multi-resource-rgd
+    kro.run/source-resource-graph-definition-name: check-multi-resource-rgd
```

Comment:
I agree with @jakobmoellerdev here, let's have a test that shows path1 and path2 (i guess both rgd and source-rgd) label will have to exist?

Reply:
done